pkg/api/v1/config.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2021 The TestGrid Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/GoogleCloudPlatform/testgrid/config"
+	pb "github.com/GoogleCloudPlatform/testgrid/pb/api/v1"
```

Comment:
apipb tends to be the more common pattern

Reply:
Fixed
/hold cancel